APIHelper.h

```diff
@@ -82,6 +82,13 @@ class Ref
 		return *this;
 	}
 
+	void Release()
+	{
+		if (m_Ref && !m_Ref->Release())
```

Comment:
I know this an old code, but it seems we don't validate if Release is called too much. Meaning the counter might become negative, making this m_Ref->Release() return true. Should we care, put an assert or something?

Reply:
We null the m_Ref after each release (now, after changes), so it's self protected from excessive releases. Previously it was protected from that being private.

Reply:
hmph, okay